scripts/actions/mobskills/aerial_collision.lua

```diff
@@ -16,7 +16,9 @@ mobskillObject.onMobWeaponSkill = function(target, mob, skill)
     local info    = xi.mobskills.mobPhysicalMove(mob, target, skill, numhits, accmod, ftp, xi.mobskills.physicalTpBonus.NO_EFFECT)
     local dmg     = xi.mobskills.mobFinalAdjustments(info.dmg, mob, skill, target, xi.attackType.PHYSICAL, xi.damageType.NONE, xi.mobskills.shadowBehavior.WIPE_SHADOWS)
 
-    xi.mobskills.mobPhysicalStatusEffectMove(mob, target, skill, xi.effect.DEFENSE_DOWN, 10, 0, 30)
+    local power = 50 + math.floor(12.5 * math.floor((skill:getTP() - 1000) / 1000))
```

Comment:
Jimmy's notes says this scales inversely but this looks like a normal scale. cap is 50% according to

```
Aerial Collision defense down:  -38% ~ -50% defense. Decays every 2 ticks by 10% -> 6% -> 6% -> 4% -> 4% -> 4% and then wears off. Lower durations just end without decaying fully.
```

-38% is seemingly arbitrary and is just a function of a mob not getting exactly 1k tp to fire

We don't support the DEFP tick down in DEFENSE_DOWN yet but i'd put a comment about it

Reply:
That made me curious to do more testing on general mob skills that apply defense down, if all of them decay or if its specific to each move. I'll rework that formula.

Reply:
Adjusted that formula to go from 50 to 40 based on TP